test/Microsoft.Data.Analysis.Tests/BufferTests.cs

```diff
@@ -188,6 +188,28 @@ public void TestArrowStringColumnClone()
                 Assert.Null(clone[i]);
         }
 
+        /* Don't run tests during build as they fail, because build if build machine doesn't have enought memory
```

Comment:
@asmirnov82 are you sure this is the reason?

Can we add a flag so these still run locally?

Reply:
We should be able to do a #if and then set a flag in build definition

Reply:
@asmirnov82 can you open an issue. I don't want to block on this.

@tannergooding just said this might actually be a netfx thing? There might need to be something set to support  > 2gb.

"For the more than 2GB one, it will be allowed but will then fail on .NET Framework anyways due to GC limitations

You have to opt in with gcAllowVeryLargeObjects iirc, and that's only for 64-bit"

Reply:
@JakeRadMSFT I fixed tests using X64FactAttribute. Please see https://github.com/dotnet/machinelearning/pull/6758 and https://github.com/dotnet/machinelearning/pull/6759